src/components/Header/Header.tsx

```diff
@@ -133,7 +137,11 @@ const Header: React.FC<Props> = ({
     return isLoggedIn ? (
       <React.Fragment>
         <IconButton className={classNames(styles.iconButton, styles.actionButton)} aria-label={t('open_user_menu')} onClick={openUserMenu}>
-          {currentProfile?.avatar_url ? <ProfileCircle src={currentProfile.avatar_url} alt={currentProfile.name} /> : <AccountCircle />}
+          {profilesEnabled && currentProfile ? (
+            <ProfileCircle src={currentProfile.avatar_url} alt={currentProfile?.name ?? t('profile_icon')} />
```

Comment:
1. You don't need the optional chaining here
2. In this case, you don't want to use the nullish coalescing operator to prevent an empty alt attribute

```suggestion
            <ProfileCircle src={currentProfile.avatar_url} alt={currentProfile.name || t('profile_icon')} />
```